plugins/gocd/src/components/GoCdBuildsComponent/GoCdBuildsComponent.test.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { GoCdBuildsComponent } from './GoCdBuildsComponent';
+import { GoCdApi } from '../../api/gocdApi';
+
+let entityValue: {
+  entity: { metadata: { annotations?: { [key: string]: string } } };
+};
+
+const mockApiClient: GoCdApi = {
+  getPipelineHistory: jest.fn(async () => ({
+    _links: { next: { href: 'some-href' } },
+    pipelines: [],
+  })),
+};
+
+jest.mock('@backstage/plugin-catalog-react', () => ({
+  useEntity: () => {
```

Comment:
This can be injected using the `EntityProvider` instead, see https://github.com/backstage/backstage/blob/1a62c3fec9e33a677bf90ed4f84d6510a1fa5f44/plugins/todo/src/components/TodoList/TodoList.test.tsx#L45